.travis.yml

```diff
@@ -87,14 +89,10 @@ before_script:
     - sed -i "s/yourusernamehere/root/" wp-tests-config.php
     - sed -i "s/yourpasswordhere//" wp-tests-config.php
     - cd "/tmp/wordpress/src/wp-content/plugins/$PLUGIN_SLUG"
-    - gem install sass
-    - gem install compass
-    - nvm install 5
-    - npm install -g npm@'3.8.9'
-    - npm install -g gulp-cli
-    - npm install
+    - ./tests/load-travis.sh
 
-script: gulp $WP_TRAVISCI
+script:
+    - $WP_TRAVISCI
```

Comment:
We weren't running ms tests when that happened but need to look back. We hadn't had success getting gulp to error out with phpunit, so I'm for moving this in to get good Travis reporting sooner rather than later and come back to ms testing.